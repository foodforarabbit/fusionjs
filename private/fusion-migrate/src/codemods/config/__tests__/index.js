const path = require('path');
const loadConfig = require('../../../utils/load-config.js');
const pluginTester = require('../../../utils/codemod-test.js');

const fixtureDir = path.join(__dirname, '../__fixtures__');
const getPlugin = require('../plugin');

pluginTester(
  fixtureDir,
  getPlugin({
    keyPath: 'clients.atreyu',
    config: loadConfig(
      path.join(__dirname, '../../../__fixtures__/atreyu-config-fixture')
    ),
  })
);
