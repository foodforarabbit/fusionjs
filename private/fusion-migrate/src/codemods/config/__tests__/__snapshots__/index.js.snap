// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`update clients.atreyu config positive-match: positive-match 1`] = `
"/* Fetch the IDL files when you add new services t.uber.com/idl */

export default {
  // Enumerate what backend services you are communicating with.
  // Find services at https://infra.uberinternal.com/apps/
  // Fetch the IDL files when you add new services via [idl command](t.uber.com/idl)
  serviceNames: ['populous'], // list of downstream service names
};


      ↓ ↓ ↓ ↓ ↓ ↓

/* Fetch the IDL files when you add new services t.uber.com/idl */

export default Object.assign(
  {enabled: true, appName: 'test'},
  __DEV__ ? {development: true} : {},
  process.env.UBER_ENVIRONMENT === 'production' ? {production: true} : {}
);
"
`;
